pkg/routeagent_driver/handlers/healthchecker/healthchecker.go

```diff
@@ -300,6 +300,9 @@ func (h *controller) syncRouteAgentStatus() {
 
 func (h *controller) generateRouteAgentObject() *submarinerv1.RouteAgent {
 	return &submarinerv1.RouteAgent{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "RouteAgent",
+		},
```

Comment:
This should not be needed. We saw this issue before which was addressed by https://github.com/submariner-io/submariner/pull/3166.

Reply:
Ack, saw this error in oldest version that didn't include #3166 , will close it